AUTHORS.md

```diff
@@ -18,8 +18,26 @@ The rules for this file:
   * Please start a new section for each new year
   * Don't ever delete anything
 -->
+**2020**
+- Lily Wang \<@lilyminium\>
+- Irfan Alibay \<@IAlibay\>
+- Oliver Beckstein \<@orbeckst\>
+- Philip Loche \<@PicoCentauri\>
+
+**2021**
+- Lily Wang \<@lilyminium\>
```

Comment:
I probably wouldn't double up on each year? (at least we don't in core, I'm not sure if there's a different standard for this)

Reply:
Just do first year.

I also wouldn't backslash-escape: the file is primarily for humans.

Reply:
Removed duplicates and removed backslashes